rkt/metadata_service.go

```diff
@@ -395,12 +405,19 @@ func handleAppAnnotations(w http.ResponseWriter, r *http.Request, pm *schema.Pod
 		return
 	}
 
-	w.Header().Add("Content-Type", "text/plain; charset=us-ascii")
-	w.WriteHeader(http.StatusOK)
+	anno := mergeAppAnnotations(im, pm, an)
+	out, err := anno.MarshalJSON()
 
-	for _, annot := range mergeAppAnnotations(im, pm, an) {
-		fmt.Fprintln(w, string(annot.Name))
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		fmt.Fprint(w, err.Error())
+		return
 	}
+
+	w.Header().Add("Content-Type", "application/json")
```

Comment:
I don't think that's necessary - json implies utf-8. The spec only states `ascii` for all scalar return types.